### PR TITLE
Update NET Core 3.1 Dependencies

### PR DESCRIPTION
**Pull request by javierpardollama**
- Updates 3.1 to Latest Security Patches
- Updates Testing Related Dependencies

### CODE REVIEW

**Review by eltjo-k — DISMISSED** [review]

File: changelog.md [hunk 1]
```diff
@@ -7,10 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Releases]
 
+## [1.5.1] - 2022-12-19
+### Added
```
Comment (Xelit3) [inline changelog.md:11]: Section should be **Changed** in that case

Comment (Xelit3) [reply changelog.md:11]: Related to that, I will request if possible to properly update the 1.5.0 entry with two entries:
- Added -> .net 7 support
- Changed -> current sdk's dependencies upgrade.

Many thanks @javierpardollama !!

**Review by Xelit3 — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (javierpardollama) [reply changelog.md]: Done @Xelit3

**Review by Xelit3 — APPROVED** [review]
Approved sir!! thanks @javierpardollama

**Review by GabriMC — APPROVED** [review]

**Review by eltjo-k — APPROVED** [review]

**Pull request merged**